### DISCUSSION TITLE
Automatically set audio to "teamradio" for OBCs

### DISCUSSION THREAD

**Discussion by FelixSFD** [f1multiviewer/issue-tracker#51] [Feature requests] [answered]
When I open an onboard stream, I usually like to listen to the teamradio, not the commentary, since I already have the international feed open in a different window.
It would be nice, if a player detects that it's an OBC and therefore switches to teamradio automatically.

**Comment by f1multiviewer** [comment]
Interesting, it actually does this for me, what audio channel is default for you?

**Reply by FelixSFD** [reply]
German commentary is the default audio for all channels.

**Reply by stefanverkerknl** [reply]
Dutch commentary is default for me when I open an OBC. So I have the same issue.

**Comment by JJWatMyself** [comment]
I have not experienced this issue.

**Comment by f1multiviewer** [comment, marked as answer]
This is now handled correctly!
